spyder/widgets/findinfiles.py

```diff
@@ -576,6 +612,10 @@ def activated(self, item):
             filename, lineno, colno = itemdata
             self.parent().edit_goto.emit(filename, lineno, self.search_text)
 
+    def enable_sorting(self, flag):
+        """Enable result sorting after search is complete."""
+        self.sorting_status['status'] = flag

```

Comment:
Why this has to be a dictionary instead of a property? I mean, you could simply have

    self.sorting = True/False

instead of having a method called `enable_sorting` that writes to a dict.